app/src/app/components/contents/common/FilterInput.tsx

```diff
@@ -3,13 +3,22 @@ import { Dispatch, SetStateAction, useCallback, useEffect, useRef } from "react"
 import { Input } from "../../Base/Input";
 import { Button } from "../../Base/Button";
 import { X } from "lucide-react";
+import { cn } from "../../../../lib/cn";
 
 interface FilterByNameProps {
   setFilter: Dispatch<SetStateAction<string>>;
   postFilterAction?: () => void;
   placeholder: string;
+  inputClassNames?: string;
```

Comment:
I won't be a stickler about it but maybe it would be more correct for this to be a list not a string. And it's how the interface of the `cn` function is designed.

Reply:
yeah there wont be a point in passing in each class name and spreading because I don't want to merge each one (if needed).... the base components in shadcn all put in multiple classnames into a string as well!!